zip-draft-XXXX.rst

```diff
@@ -0,0 +1,289 @@
+::
+
+  ZIP: Unassigned
+  Title: Lockbox for Decentralized Grants Allocation
+  Owners: Kris Nuttycombe <kris@nutty.land>
+          Jason McGee <aquietinvestor@gmail.com>
+  Original-Authors: Skylar Saveland <skylar@free2z.com>
+  Credits: @daira
+           @str4d
+  Status: Draft
+  Category: Consensus
+  Created: 2024-06-26
+  License: MIT
+  Pull-Request: <https://github.com/zcash/zips/pull/866>
+
+
+Terminology
+===========
+
+The key words "MUST", "REQUIRED", "MUST NOT", "SHOULD", and "MAY" in this
+document are to be interpreted as described in BCP 14 [#BCP14]_ when, and only
+when, they appear in all capitals.
+
+.. {Avoid duplicating definitions from other ZIPs. Instead use wording like this:}
+
+.. The terms "Mainnet" and "Testnet" in this document are to be interpreted as
+.. defined in the Zcash protocol specification [#protocol-networks]_.
+
+.. The term "full validator" in this document is to be interpreted as defined in
+.. the Zcash protocol specification [#protocol-blockchain]_.
+
+.. The terms below are to be interpreted as follows:
+
+.. {Term to be defined}
+..   {Definition.}
+.. {Another term}
+..   {Definition.}
+
+
+.. |percentage| replace:: 20%
```

Comment:
Other than ZIP metadata, this is the only change in this ZIP draft relative to #858. As such, all of the comments on #858 are incorporated here, and it is likely that instead of having two ZIP drafts that are identical bar a single number, the ZIP editors will structure this as a single ZIP with two alternatives for this parameter.